riptide-core/src/main/java/org/zalando/riptide/Rest.java

```diff
@@ -120,30 +124,36 @@ public final Requester trace(final URI url) {
 
     private Requester execute(final HttpMethod method, final String uriTemplate,
             final Object... uriVariables) {
-        return execute(method, uriTemplateHandler.expand(uriTemplate, uriVariables));
+        return new ListenableFutureRequester(method, fromUriString(uriTemplate), uriVariables);
```

Comment:
The naming is bad. Actually, I have the feeling the methods of this class should be integrated into the Requester itself.